src/utils/log/FeatureLog.ts

```diff
@@ -16,7 +22,9 @@ type TFeatureLogFlag = {
 const FeatureLogFlag: TFeatureLogFlag = {
   useSimpleWebsocketHook: false,
   useOneSignalSubscribeToCommunityHooks: false,
-  oneSignalUtils: true
+  oneSignalUtils: false,
+  navigation: false,
+  firebaseUtils: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are adding new feature log flags for `navigation` and `firebaseUtils`, and changing the value of `oneSignalUtils` from `true` to `false`. This seems fine as long as it's intentional and you're aware that `oneSignalUtils` logging will be turned off by default. If this is not the intended behavior, you should revert the change.
```diff
-  oneSignalUtils: true
+  oneSignalUtils: false,
+  navigation: false,
+  firebaseUtils: false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->